packages/loaders/src/Dots.example.md

```diff
@@ -0,0 +1,179 @@
+The `<Dots />` component includes several accessibility
+and usability features:
+
+- Inherits the `font-size` and `color` properties
+  from it's parent element and additionally allows local overrides
+- Delays the initial render of the component to reduce UI shifting
+  in normal loading conditions
+- Includes accessibility attributes to identify as an `indeterminate progress bar`
+
+### Basic Usage
+
+```jsx
+const { zdColorBlue500 } = require('@zendeskgarden/css-variables');
+
+<Dots color={zdColorBlue500} size="50px" />;
+```
+
+### Loading Button
```

Comment:
These were comments that I brought up yesterday, but we didn't get it all lined out:

> abrupt resizing of the button (should possibly retain the width of it's text)

We can make this match width in the example, but with localized text this should never be something that is prescribed. Consumers would have to understand the specific length of all localized text to make sure the width is correct.

> the closer I look, the loader seems to sit a bit high and sizing might be a bit off

Since this animation has a "low" center of gravity this will have to be customized for every usage within another element. Maybe we could modify the viewport to create a square sizing with the bottom padding already added, but that would cause many problems since this is also being prescribed as a baseline animation in some areas.

Reply:
If you put the text in a `<span>`, you can set `visibility: hidden` to retain button size. I'm less concerned about the vertical positioning – that's just a nit.

Reply:
We should definitely match the width of the button on loader display. The jumping is terribly distracting, and it's not the intention of the design. 

Also, I agree that that loader looks a bit high in the container. And in fact, precise measurements show that it is:

![image](https://user-images.githubusercontent.com/1230973/44434308-5026db00-a55f-11e8-948d-50092e42425c.png)

@austin94 we can pair on where it should be in the button—a few pixels down at least.